addon/components/sl-menu.js

```diff
@@ -338,11 +341,11 @@ export default Ember.Component.extend({
     children: null,
 
     /**
-     * Additional class names to be added to the classNames attribute
+     * Additional class string to be added to the element's class attribute
      *
-     * @type {String[]}
+     * @type {String}
      */
-    extraClassNames: [],
+    className: '',
```

Comment:
Change this name to something not so similar to the "classNames" property that already exists on components.